src/SDKs/KeyVault/Management/Management.KeyVault/Generated/Models/StoragePermissions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.17.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.KeyVault.Models
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for StoragePermissions.
+    /// </summary>
+    public static class StoragePermissions
+    {
+        public const string Get = "get";
+        public const string List = "list";
+        public const string Delete = "delete";
+        public const string Set = "set";
+        public const string Update = "update";
+        public const string Regeneratekey = "regeneratekey";
```

Comment:
I don't think casing matters here, besides I followed the existing model: https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/KeyVault/Management/Management.KeyVault/Generated/Models/CertificatePermissions.cs